src/c++/perf_analyzer/main.cc

```diff
@@ -26,12 +26,15 @@
 
 #include <getopt.h>
 #include <signal.h>
+
```

Comment:
Are we using a different formatter?

Reply:
clang-format version 10.0 was likely used (it's the default version installed in ubuntu 20.04)--it adds the gaps like here

since the [format.py](https://github.com/triton-inference-server/common/blob/main/tools/format.py#L68) in our GitHub org suggests to use clang-format version 6.0, i've installed that (`apt update ; apt install -y clang-format-6.0`) and reformatted, and these gaps are no longer added 👍